mito-ai/src/Extensions/AiChat/ChatHistoryManager.tsx

```diff
@@ -299,13 +339,14 @@ export class ChatHistoryManager {
     }
 
     addAIMessageFromAgentResponse(agentResponse: AgentResponse): void {
+        const deduplicatedAgentResponse = this.deduplicateAssumptions(agentResponse);
```

Comment:
Small nitpick but maybe we can just called this `agentResponseWithDeduplicatedAssumptions` just so it's more clear what is actually deduplicated.